tests/test_hear_keyboard_interrupt.py

```
@@ -0,0 +1,85 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__module__ = """tests"""
+
+try:
+	try:
+		import context
+	except Exception as ImportErr:  # pragma: no branch
```

Ruff

Local variable `ImportErr` is assigned to but never used

Remove assignment to unused variable `ImportErr`

(F841)